hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java

```diff
@@ -791,15 +791,33 @@ public short getMinReplication() {
     return minReplication;
   }
 
+  public short getMinStorageNum(BlockInfo block) {
```

Comment:
@zz12341  Why we want these changes? I don't see it in the original patch [here](https://github.com/apache/hadoop/pull/4410/files).

Reply:
I was originally trying to make it in sync with what trunk branch is doing: https://github.com/apache/hadoop/pull/4410/files#diff-305ecf45a0f0708849b5e3c0d21a56c681db3a1497e52a19ef24939278dc99feL1922-R1926

Let me revert this change